raco/language/myrialang.py

```diff
@@ -781,6 +810,9 @@ def __init__(self, input):
     def num_tuples(self):
         return self.input.num_tuples()
 
+    def partitioning(self):
```

Comment:
Shouldn't `partitioning` for the Consumer side of an exchange operator reflect the partitioning it imposes on its output (i.e., the actual partitioning of tuples on the worker where it's running)?